dialogflow/fulfillment-weather-python
# Node Detail View

This branch improves the graph visualization (larger ellipses with centered text, edge labels) and adds a detail view for selected nodes.

To test:
1. Ask the Exprt: "How does SHI ship to Europe".
2. Observe that you can drag the graph nodes and that selecting "SHI" shows a number of properties of the SHI node.
